docs/maintenance/upgrade.md

```diff
@@ -21,10 +21,17 @@ To upgrade Codacy to the latest stable version:
 
 3.  Review the values stored in the file `codacy.yaml`, making any changes if necessary.
 
-4.  Perform the upgrade using the values stored in the file:
+4.  Get the latest changes from our helm repository
+
+    ```bash
+    helm repo update
```

Comment:
To simplify, we could simply include this command as the first line of the code block in the next step (similar to what we have in the initial installation instructions https://docs.codacy.com/chart/#helm-upgrade).